testing/test-app/app/components/table.hbs

```diff
@@ -0,0 +1,23 @@
+<table role="grid" {{aria-grid (with-default @isMac false)}} ...attributes>
```

Comment:
Turned out the tests were passing locally but failing in CI because I forgot this `@isMac` default 🤦‍♀️